releasenotes/notes/apm-disable-sublayers-62ad93870087595f.yaml

```diff
@@ -0,0 +1,13 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    APM: Sublayer metrics (trace.<SPAN_NAME>.duration and derivatives) computation
```

Comment:
I have two questions here:

1. Why would anyone want to disable them?
2. Feature flags are generally experimental features. Why do you want them to show up in the changelog?